lib/_algoliaAgent.ts

```diff
@@ -0,0 +1,10 @@
+import { version } from "../package.json";
+
+export const DEFAULT_ALGOLIA_AGENT = `Algolia insights for JavaScript (${version})`;
+
+export function addAlgoliaAgent(algoliaAgent) {
+  if (this._ua.indexOf(`;${algoliaAgent}`) === -1) {
+    this._ua += `;${algoliaAgent}`;
```

Comment:
@samouss separated this with spaces in the client

Reply:
I look for it as I wasn't sure where the spaces were but couldn't find it.
https://github.com/algolia/algoliasearch-client-javascript/blob/develop/src/AlgoliaSearchCore.js#L173-L177
Am I looking at the right branch?

Reply:
I think it’s in Robert’s branch still

Reply:
thanks! it's fixed here: 2546385